tests/rbd_mirror/test_rbd_mirror_reconfig.py

```diff
@@ -0,0 +1,113 @@
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_mirror_reconfig(rbd_mirror, pool_type, **kw):
+    """Method to validate reconfiguring mirror on existing mirror cluster
+    got prevented or not.
+
+    Args:
+        rbd_mirror: Object
+        pool_type: Replicated or EC pool
+        **kw: test data
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+    try:
+        config = kw.get("config")
+        mirror1 = rbd_mirror.get("mirror1")
+        mirror2 = rbd_mirror.get("mirror2")
+        pool = config[pool_type]["pool"]
+        mode = config[pool_type]["mode"]
+        primary_cluster = kw.get("ceph_cluster")
+        primary_cluster_name = primary_cluster.name
+        mirror1.bootstrap_peers(poolname=pool, cluster_name=primary_cluster_name)
+        # Attempt creating mirror in same cluster this should be prevented
+        out, err = mirror1.import_bootstrap(
```

Comment:
[**optional**] can we log the actual output for import_bootstrap command, to cross verify the message we are logging.

Reply:
IMO we don't need to log output as it's negative scenario command is failing, it's in `err` output been cross verified in line 32 Hence no need of output log.